packages/visx-drag/src/useDrag.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useCallback } from 'react';
+import { localPoint } from '@visx/event';
+import useStateWithCallback from './util/useStateWithCallback';
+
+type MouseOrTouchEvent = React.MouseEvent | React.TouchEvent;
```

Comment:
Could add `React.PointerEvent`. Was added in React 16.4 https://reactjs.org/blog/2018/05/23/react-v-16-4.html#pointer-events